operators/admin/olm-cs-podsched.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: ASSEMBLY
+[id="olm-cs-podsched"]
+= Catalog source pod scheduling
+include::_attributes/common-attributes.adoc[]
+:context: olm-cs-podsched
+
+toc::[]
+
+When an Operator Lifecycle Manager (OLM) catalog source of source type `grpc` defines a `spec.image`, the Catalog Operator creates a pod that serves the defined image content. By default, this pod defines the following in its spec:
+
+* Only the `kubernetes.io/os=linux` node selector
+* No priority class name
+* No tolerations
+
+As an administrator, you can override these values by modifying fields in the `CatalogSource` object's optional `spec.grpcPodConfig` section.
+
+[role="_additional-resources"]
```

Comment:
I think you need unique ids for all of the additional resources. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
That's for when it's a `==` heading, whereas these `.` ones are supposed to look like they are at the end of a module/section (other than the intro matter which doesn't have its own module, but is sorta like a section).

Reply:
My mistake, sorry Alex.